assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -80,12 +85,104 @@ export const actions = {
 			type: RECEIVE_ACCOUNTS_PROPERTIES_PROFILES_FAILED,
 		};
 	},
+
+	/**
+	 * Creates a new Analytics account.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} args              Argument params.
+	 * @param {string} args.accountName  Google Analytics account name.
+	 * @param {string} args.propertyName Google Analytics property name.
+	 * @param {string} args.profileName  Google Analytics profile name.
+	 * @param {string} args.timezone     Google Analytics timezone.
+	 * @return {Function} Generator function action.
+	 */
+	*createAccount( { accountName, propertyName, profileName, timezone } ) {
+		invariant( accountName, 'accountName is required.' );
+		invariant( propertyName, 'propertyName is required.' );
+		invariant( profileName, 'profileName is required.' );
+		invariant( timezone, 'timezone is required.' );
+
+		const accountTicket = yield actions.fetchCreateAccount( { accountName, propertyName, profileName, timezone } );
+
+		return actions.receiveCreateAccount( { accountTicket } );
+	},
+
+	*fetchCreateAccount( { accountName, propertyName, profileName, timezone } ) {
+		let response;
+
+		yield {
+			payload: {},
+			type: START_FETCH_CREATE_ACCOUNT,
+		};
+
+		try {
+			response = yield {
+				payload: { accountName, propertyName, profileName, timezone },
+				type: FETCH_CREATE_ACCOUNT,
+			};
+
+			yield {
+				payload: {},
+				type: FINISH_FETCH_CREATE_ACCOUNT,
+			};
+		} catch ( error ) {
+			return actions.receiveCreateAccountFailed( { accountName, error } );
+		}
+		return response;
+	},
+
+	/**
+	 * Adds an account ticket to process.
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @param {Object} args               Argument params.
+	 * @param {Object} args.accountTicket Google Analytics create account ticket object.
+	 */
+	receiveCreateAccount( { accountTicket } ) {
```

Comment:
No need to wrap this in an object, this could become `receiveCreateAccount( accountTicket )` for simpler usage (the `accountTicket` is also directly the above `response` from the API).

Reply:
ok, changing. Using this more by convention.

Reply:
Changed in https://github.com/google/site-kit-wp/pull/1390/commits/d4f1bc132be3e220d3b73747b1d7ed8b7f6b9c99